q2_feature_table/tests/filter/test_filter_features.py

```diff
@@ -190,6 +217,16 @@ def test_where(self):
                          ['S1', 'S2', 'S3'])
         self.assertEqual(actual, expected)
 
+    def test_allow_empty_table(self):
```

Comment:
You can remove this test since we're testing the True case for this param with each of the above test cases!